ecs-cli/modules/utils/compose/convert_task_definition.go

```diff
@@ -125,37 +127,32 @@ func ConvertToTaskDefinition(context *project.Context, volumeConfigs map[string]
 		taskRoleArn = taskDefParams.taskRoleArn
 	}
 
+	// TODO: Refactor when Volumes added to top level project
 	volumes, err := ConvertToVolumes(volumeConfigs)
 	if err != nil {
 		return nil, err
 	}
 
 	// Create containerDefinitions
 	containerDefinitions := []*ecs.ContainerDefinition{}
-	for _, name := range serviceConfigs.Keys() {
-		serviceConfig, ok := serviceConfigs.Get(name)
-		if !ok {
-			return nil, fmt.Errorf("Couldn't get service with name=[%s]", name)
-		}
-		logUnsupportedServiceConfigFields(name, serviceConfig)
-		containerDef := &ecs.ContainerDefinition{
-			Name: aws.String(name),
-		}
 
+	for _, containerConfig := range containerConfigs {
+		// logUnsupportedServiceConfigFields(name, serviceConfig) // TODO switch this to use ContainerConfig
```

Comment:
re: the TODO - we won't have visibility into unsupported fields by the time we get the containerConfigs, will we? we'd need to do that in the parsing step(s) [this might be an old TODO]

Reply:
Right, this needs to be moved into the `parseCompose` step but I figured I'd defer on that since it touches both v1v2 and v3.

Reply:
+1